usagov/vote-gov-drupal
# Vote-2887: Outside US info card

## Jira ticket

[Vote-2887](https://cm-jira.usa.gov/browse/VOTE-2887)

## Description

Setting up config for the outside of us info card on state pages.

![Screenshot 2024-10-03 at 2 58 38 PM](https://github.com/user-attachments/assets/3be63804-18f5-4d88-bb36-448b274bcd3d)


## Deployment and testing

### Post-deploy steps

1. run `lando retune`

### QA/Testing instructions

1. Log into site and create a new state page, go to the `State Display` content tab 
2. Scroll to the `Military and overseas registration` and add text 
3. Visit page and check that it is matching the design in the figma 

## Checklist for the Developer




## Checklist for the Peer Reviewers
